src/helpers/helpers.js

```diff
@@ -0,0 +1,3 @@
+export const generateId = () => Math.floor(Math.random() * 1000000)
```

Comment:
I love the arrow functions so much. :) 
As well it's just one line instead three.